src/collections/_documentation/clients/go/context.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Context
+sidebar_order: 3
+---
+
+All of the `Capture*` functions accept an additional argument for passing a `map` of tags as the second argument and context data as remaining ones.
+Tags in Sentry help to categories and give you more information about the errors that happened, where other context data is more "environment" specific.
+(NOTE: direct low-level `Capture` call accepts second argument only - tags, without additional interfaces)
+
+For example:
+
+```go
+raven.CaptureError(err, map[string]string{"browser": "Firefox"}, &raven.Http{
+  Method: "GET",
+  URL: "https://example.com/raven-go"
+})
+```
+
+This data can be either passed directly to `Capture*` calls or configured globally using functions listed below, if it should be applied to all future events.
```

Comment:
Tried fixing passive voice to active voice. Not sure what subject is performing the action. I'm assuming the user? Here's my edit, "This data can be either passed directly to Capture* calls or configured globally using functions listed below, if you decide to apply it to all future events."